src/store/selectors.js

```diff
@@ -0,0 +1,4 @@
+const tokenSelector = (store) => store.token.token;
+const themeSelector = (store) => store.theme.theme;
```

Comment:
as i see, state structure can be simplified to `{ theme: string; token: string }`